openhtf/io/user_input.py

```diff
@@ -63,7 +63,8 @@ def __init__(self):
     self._response = None
     self._cond = threading.Condition()
 
-  def DisplayPrompt(self, message, text_input=False):
+  def DisplayPrompt(self, message, text_input=False,
+                    timeout=FLAGS.prompt_timeout_s):
```

Comment:
It's generally bad to use mutable types as default values (there's a good explanation of why here: http://docs.python-guide.org/en/latest/writing/gotchas/), better to default to None and then check for None in the code.